src/Rules/NotPresent.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Saritasa\Laravel\Validation\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+use Saritasa\Laravel\Validation\IRule;
+
+/**
+ * Not present validation for a field.
+ */
+class NotPresent implements IRule, Rule
+{
+    /**
+     * The name of the rule.
+     */
+    protected $rule = 'not_present';
+
+    /**
+     * Convert the rule to a validation string.
+     *
+     * @return string
+     */
+    public function __toString(): string
+    {
+        return $this->rule;
+    }
+
+    /**
+     * Check whether attribute value is set or not.
+     *
+     * @param  string  $attribute Attribute name
+     * @param  mixed  $value Attribute value
+     * @return boolean
+     */
+    public function passes($attribute, $value)
+    {
+        return is_null($value);
```

Comment:
Rule described as 'attribute not present' but checked as 'attribute is empty'. According to existing laravel's 'present' rule we should check that attribute should not be presented in array of validated attributes. We shoud change name of this rule to 'empty' or change code to check that attribute not presented in array of validated attributes.

Reply:
As I know, If the field is not present, the value is always null. 
I added a function to test this case and mentioned you.

Reply:
Let's imagine that we want to take all parameters from request and update model according to this parameters.

For example we write rule for field 'age' as 'not_present', but it is presented (but null) and after updating model by all request fields age will be updated to null